app/controllers/ReleasedController.scala

```diff
@@ -20,18 +21,21 @@ class ReleasedController @Inject()(
   def getMovies = Action.async { implicit request: Request[AnyContent] =>
     releasedServices.getMovies.map { movies =>
 //   Ok(movies.toString())
-      Ok(views.html.movie(movies))
+      Ok(views.html.releasedmovie(movies))
 
     }
   }
 
-  def ReleasedMovieInfo(id: String) = Action.async { implicit request: Request[AnyContent] =>
+  def getOnlyMoviesAndScreenings: Action[AnyContent] = Action.async { implicit request: Request[AnyContent] =>
     releasedServices.getMovies.map { movies =>
-
-
-      Ok(views.html.movieInfo(movies.filter(movie => id == movie._id.toString()).head))
+      movies.map(movie => movie.title -> movie.screenings)
+    }.map { movies =>
+      Ok(movies.toString())
+    }
+  }
+  def releasedMovieInfo(id: String) = Action.async { implicit request: Request[AnyContent] =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method takes a value that is controlled by the client](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5277123)